tests/behavioural/features/steps/etag.py

```diff
@@ -12,14 +12,14 @@ def step_impl_etag_should_be_sent_with_draft(context):
 
 @given("an etag is requested with an empty value")
 @when("an etag is requested with an empty value")
-def step_impl_etag_is_requested(context):
+def step_impl_etag_is_requested_with_empty_value(context):
     """specify an etag but do not give it a value"""
     context.bdd_helper.headers["Etag"] = ""
 
 
 @given("an etag is requested with a value of '{etag}'")
 @when("an etag is requested with a value of '{etag}'")
-def step_impl_etag_is_requested(context, etag):
+def step_impl_etag_is_requested_with_specific_value(context, etag):
     """specify an etag ofa specificvalue"""
```

Comment:
Mind sorting out the spacing here while you're there?

Reply:
done